header.php

```diff
@@ -30,8 +30,12 @@
 				<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
 			<?php else : ?>
 				<p class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></p>
+			<?php endif;
+
+			$description = get_bloginfo( 'description', 'display' );
+			if ( $description || is_customize_preview() ) : ?>
+				<p class="site-description"><?php echo $description; ?></p>
```

Comment:
Would `<?php echo esc_html( $description ); ?>` be better? Having trouble keeping up with all the changing opinions around escaping, not escaping enough, and escaping too much 'round these parts.

FWIW here's what goes into all my themes:

```php
								<?php
								$description = get_bloginfo( 'description', 'display' );
								if ( ! empty( $description ) || is_customize_preview() ) : ?>
									<p class="site-description"><?php echo esc_html( $description ); ?></p>
								<?php endif; ?>
```

Pretty much the same as what you're proposing.